lib/collection.js

```diff
@@ -175,6 +175,7 @@ const BookshelfCollection = (module.exports = CollectionBase.extend(
           // next step of the collection. Since the `columns` are only relevant to the current
           // level, ensure those are omitted from the options.
           .tap(function(response) {
+            if (!response || response.length === 0) return;
```

Comment:
It's weird, I don't see anywhere in their code where they ever handled the option.require flag. That being said, it would be good when we have time to get rid of this package, it looks like it hasn't been contributed to in over a year.